examples/python/ex_stat.py

```diff
@@ -0,0 +1,125 @@
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# ex_stat.py
+#      This is an example demonstrating how to query database statistics.
+
+import os
+from wiredtiger import wiredtiger_open,WIREDTIGER_VERSION_STRING
+from _wiredtiger import WT_STAT_DSRC_BLOCK_SIZE,WT_STAT_DSRC_BLOCK_CHECKPOINT_SIZE,WT_STAT_DSRC_CURSOR_INSERT_BYTES,WT_STAT_DSRC_CURSOR_REMOVE_BYTES,WT_STAT_DSRC_CURSOR_UPDATE_BYTES,WT_STAT_DSRC_CACHE_BYTES_WRITE,WT_STAT_DSRC_BTREE_OVERFLOW
+
+def main():
+	# Create a clean test directory for this run of the test program
+	# if it does not already exist
+	if not os.path.exists('WT_TEST'):
+		os.makedirs('WT_TEST')
+	# Connect to the database and open a session
+	conn = wiredtiger_open('WT_TEST', 'create,statistics=(all)')
+	session = conn.open_session()
+	
+	# Create a simple table
+	session.create('table:access', 'key_format=S,value_format=S')
+	
+	# Open a cursor and insert a record
+	cursor = session.open_cursor('table:access', None)
+	
+	cursor.set_key('key')
+	cursor.set_value('value')
+	cursor.insert()
+	cursor.close()
+	
+	session.checkpoint()
+	print WIREDTIGER_VERSION_STRING
+	print_database_stats(session)
+	print_file_stats(session)
+	print_overflow_pages(session)
+	print_derived_stats(session)
+	conn.close()
+
+def print_database_stats(session):
+	try:
+		statcursor = session.open_cursor("statistics:")
+	except:
+		return
+	print_cursor(statcursor)
+	statcursor.close()
+
+def print_file_stats(session):
+	try:
+		fstatcursor = session.open_cursor("statistics:table:access")
+	except:
+		return
+	print_cursor(fstatcursor)
+	fstatcursor.close()
+
+def print_overflow_pages(session):
+	try:
+		ostatcursor = session.open_cursor("statistics:table:access")
+	except:
+		return
+	ostatcursor.set_key(WT_STAT_DSRC_BTREE_OVERFLOW)
+	ostatcursor.search()
+	val = ostatcursor.get_value()
+	if val != 0 :
+		print str(val[0]) + '=' + str(val[1])
+	ostatcursor.close()
+
+def print_derived_stats(session):
+	try:
+		dstatcursor = session.open_cursor("statistics:table:access")
+	except:
+		return
+	ckpt_size = get_stat(dstatcursor, WT_STAT_DSRC_BLOCK_CHECKPOINT_SIZE)
+	file_size = get_stat(dstatcursor, WT_STAT_DSRC_BLOCK_SIZE)
+	percent = 0
+	if file_size != 0 :
+		percent = 100 * ((float(file_size) - float(ckpt_size)) / float(file_size))
+	print "Table is %" + str(percent) + " fragmented"
+
+	app_insert = int(get_stat(dstatcursor, WT_STAT_DSRC_CURSOR_INSERT_BYTES))
```

Comment:
I think we can get rid of get_stat().  In Python we can use abbreviated `dstatcursor[WT_STAT_DSRC_CURSOR_INSERT_BYTES]` syntax.